.travis.yml

```diff
@@ -1,11 +1,17 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
+  matrix:
+    
+    - CONDA_PY=27
+    - CONDA_PY=34
+    - CONDA_PY=35
```

Comment:
So this is definitely right based on looking at the recipe. However, am a little worried it wasn't already included. Could you please give a build number bump on top of this too to get new packages, @jschueller ? We might need to pull the bad packages.

cc @pelson